tests/unit/test_sql_sensor.py

```diff
@@ -0,0 +1,155 @@
+import unittest
+from unittest import mock
+
+from airflow.exceptions import AirflowException
+from airflow_clickhouse_plugin import ClickHouseSqlSensor
+
+
+class ClickHouseSqlSensorTestCase(unittest.TestCase):
+    @mock.patch(
+        'airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor.ClickHouseHook'
+    )
+    def test_clickhouse_sql_sensor_poke(self, mock_hook):
+        op = ClickHouseSqlSensor(task_id='sql_sensor_check', sql='SELECT 1',)
+
+        mock_get_records = mock_hook().get_records
```

Comment:
Does not `unittest.mock` convert all the methods into `Mock`s too?

Reply:
BTW you can patch `'airflow_clickhouse_plugin.sensors.clickhouse_sql_sensor.ClickHouseHook.get_records'` directly